plugins/plugin-hosted/codenvy-machine-hosted/src/main/java/com/codenvy/machine/RemoteDockerNode.java

```diff
@@ -51,6 +52,10 @@
     private final String                         nodeHost;
     private final String                         nodeIp;
 
+    private volatile boolean isBound;
```

Comment:
It won't be exposed as API. Operating with one enumerated status variable will be better for code readability and understanding. And in the same time it will reduce mistakes made in future. And what about cohesion, those two methods will be connected using one variable instead of three.